.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Cortex Debug",
+            "cwd": "${workspaceFolder}",
+            "executable": "${workspaceFolder}/var/aircrafts/CobraV2/ap/obj/ap.elf",
```

Comment:
Nice, never used this. But the executable is especially for your aircraft, so maybe we need to leave this out or fix it another way.
Maybe some information on how to set this up correctly would be really nice and help a lot of people.

Reply:
This PR is still a bit messy, I will remove this file before merging.
I started a VSCode plugin for paparazzi, but its far from being useful yet...